tests/analysis/test_transport_mode_identification.py

```diff
@@ -30,9 +26,8 @@ def test_simple_coarse_identification_no_crs(self):
         tpls_file = os.path.join('tests', 'data', 'triplegs_transport_mode_identification.csv')
         tpls = ti.read_triplegs_csv(tpls_file, sep=';', index_col="id")
 
-        with pytest.warns(UserWarning, 
-                          match='Your data is not projected. WGS84 is assumed and for length ' + 
-                          'calculation the haversine distance is used'):
+        with pytest.warns(UserWarning, match='Your data is not projected. WGS84 is assumed and for length calculation'
+                                             ' the haversine distance is used'):
```

Comment:
I just changed this in the process because I had trouble getting this test to work. In the end the problem was something else